networking/synthetic/stac-prepare-nodes.py

```diff
@@ -82,30 +100,18 @@ def parse_args():
                         dest='iface_name',
                         help='interace name on nodes through which stac test traffic will run')
 
-    parser.add_argument('-p',
-                        '--tracing',
-                        required=False,
-                        dest='pbd_tracing',
-                        help='profile the app with pdb tracing')
-
     return parser.parse_args()
 
 
-def main():
+if __name__ == '__main__':
     args = parse_args()
     nettest = NetworkTest('stac-prepare-nodes.yaml')
     for node in args.nodes:
-        if args.pbd_tracing:
-            import pdb
-            pdb.set_trace()
-    	patch_node_oir(args.api_server_url, node)
-    	nettest.add_stac_node(node)
+        patch_node_oir(args.api_server_url, node)
```

Comment:
We have to be careful continuing to use OIR.  They're replaced with ER "extended resources" in 1.8, and in fact we can easily use node affinity now and resource classes if/when those get in.

Reply:
+1